src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/Example1.java

```diff
@@ -0,0 +1,32 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MethodLength"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.sizes.methodlength;
+
+// xdoc section -- start
+public class Example1 {
```

Comment:
This entire example is very hard to read with all the different comments. Please make one good method, and one method that has a violation. Please make some reasonable, simple code in each that is easy to read. Then also have one good constructor and one with a violation.

Then also declare two records, one with a good compact constructor, and one with a compact constructor that has a violation. If we need to reduce max to 3, this is also fine. This will be a good demonstration of the default check properties.

Reply:
done